lib/hanami/model/sql/entity/schema.rb

```diff
@@ -38,6 +38,24 @@ def initialize(registry, relation, mapping)
             freeze
           end
 
+          # Process attributes
+          #
+          # @param attributes [#to_hash] the attributes hash
+          #
+          # @raise [TypeError] if the process fails
+          #
+          # @since 1.0.1
+          # @api private
+          def call(attributes)
+            Utils::Hash.new(
+              schema.call(attributes)
+            ).deep_symbolize!
```

Comment:
Hi all, I'm suffering with this call to deep_symbolize! right now. Nothing that I do solve this issue and I'm forcing myself to `stringify!` on consumers of my entitties.